### PR TITLE
(Re)Fix two Coverity-reported defects

### PR DESCRIPTION
**Pull request by ejoerns**
* Fixes #871 
* Fixes 59653ab7

### CODE REVIEW

File: src/stats.c [hunk 1]
```diff
@@ -6,7 +6,7 @@ void r_stats_init(struct RaucStats *stats)
 
 	memset(stats, 0, sizeof(*stats));
 
-	stats->min = G_MAXDOUBLE;
+	stats->max = G_MAXDOUBLE;
 	stats->min = G_MINDOUBLE;
```
Comment (jluebbe) [inline src/stats.c:10]: This must be the other way around. If `max` is already `G_MAXDOUBLE`, it can't change anymore.

**Review by jluebbe — CHANGES_REQUESTED** [review]

File: src/stats.c (hunk 1 above)
Comment (ejoerns) [reply src/stats.c:10]: Indeed, now both are broken. Will fix.

**Review by jluebbe — APPROVED** [review]

File: src/stats.c [hunk 2]
```diff
@@ -7,7 +7,7 @@ void r_stats_init(struct RaucStats *stats)
 	memset(stats, 0, sizeof(*stats));
 
 	stats->min = G_MAXDOUBLE;
-	stats->min = G_MINDOUBLE;
+	stats->max = G_MINDOUBLE;
```
Comment (a3f) [inline src/stats.c:10]: Is that correct? at the very least a comment explaining why min = MAX and max = MIN might be in order.

Comment (a3f) [reply src/stats.c:10]: Just saw @jluebbe comment above. Still some comment in the code would be nice.

Comment (jluebbe) [reply src/stats.c:10]: These are initial values for incremental comparison. If they were the reversed, they could never change.

Comment (ejoerns) [reply src/stats.c:10]: There is hardly any documentation of these functions at all. But iirc I did some for me during the review.
If I find it, I'll submit :smirk:

**Pull request merged**